media/js/mozorg/plugincheck-update.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+    var body = $('body');
+
+    if (client._isFirefoxDesktop) {
+        client.getFirefoxDetails(function(details) {
+            if (details.isUpToDate) {
```

Comment:
After discussion on Slack, consensus seems to be that we should avoid calling out ESR in general, since they are unlikely to be able to update anyway.